apps/server/src/modules/deletion/uc/builder/deletion-request-log-response.builder.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { DeletionDomainModel } from '../../domain/types/deletion-domain-model.enum';
```

Comment:
Please shorten this import (export this enum via barrel file from the `types` dir.

Reply:
done